src/flows/components/panel/FlowPanel.tsx

```diff
@@ -145,6 +150,12 @@ const FlowPanelHeader: FC<HeaderProps> = ({
             </FeatureFlag>
             <PanelVisibilityToggle id={id} />
             <RemovePanelButton id={id} />
+            {isFlagEnabled('CopyClientCodeToClipboard') && (
+              <CopyToClipboardProvider>
+                <CopyToClipboardButton />
+                <PanelQueryOverlay panelId={id} />
+              </CopyToClipboardProvider>
```

Comment:
Use the popupcontext for this instead. See the exporttodashboard and exporttotask examples in src/flows.